examples/open_deep_research/run.py

```diff
@@ -11,14 +11,14 @@
     FindNextTool,
     PageDownTool,
     PageUpTool,
-    SearchInformationTool,
```

Comment:
Maybe we should remove if from run_gaia as well and eventually delete the `SearchInformationTool` class if it is no longer used.